coordinator-service/src/ceremony.ts

```diff
@@ -1,10 +1,12 @@
 export interface ChunkContributionMetadata {
     contributedTime: string
     contributedLockHolderTime: string
-    contributedSignature: string
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    contributedData: any
```

Comment:
I was originally stuffing stuff in the "metadata objects" that didn't directly impact the ceremony coordination.  Since you're using this data to coordinate the ceremony, it's a little bit of a departure from that (completely undocumented) convention.  Up to you if you want to move these fields into the `ChunkContribution` or not.

Reply:
That's a great point, I'll change it.